pkg/controller/linstorsatelliteset/linstorsatelliteset_controller.go

```diff
@@ -1331,6 +1337,67 @@ func (r *ReconcileLinstorSatelliteSet) controllerReachable(ctx context.Context,
 	return err
 }
 
+// removeDanglingSatellites removes satellites that were registered by the operator and are no longer present.
+func (r *ReconcileLinstorSatelliteSet) removeDanglingSatellites(ctx context.Context, linstorClient *lc.HighLevelClient, k8sNodes []corev1.Node) error {
+	lnodes, err := linstorClient.Nodes.GetAll(ctx, &lapi.ListOpts{
+		Prop: []string{fmt.Sprintf("%s=%s", kubeSpec.LinstorRegistrationProperty, kubeSpec.Name)},
+	},
+	)
```

Comment:
This looks odd. Close them both on one line since they were both opened on one line?